modules/core/client/controllers/core.client.controller.sidenav.left.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('core')
+    .controller('SideNavLeftController', SideNavLeftController);
+
+  SideNavLeftController.$inject = ['$mdSidenav'];
+
+  function SideNavLeftController($mdSidenav) {
+    var vm = this;
+    vm.menus = menuFactory.sidenavleft.items;
```

Comment:
This won't work.  You aren't injecting menuFactory.... don't even really want it here anyway..